src/deepsparse/image_classification/README.md

```diff
@@ -151,7 +151,7 @@ Making a request:
 import requests
 
 url = 'http://0.0.0.0:5543/predict/from_files'
-path = ['golfish.jpeg', 'golfish.jpeg'] # just put the name of images in here
+path = ['goldfish.jpeg'] # just put the name of images in here
```

Comment:
I'd still keep two image strings there, so that the user can intuitively understand that they can pass multiple images at once.

Reply:
@dbogunowicz They actually cannot pass two images at once with the example server command just above as it does not pass the `--batch_size` flag with an appropriate value.

This specific doc change is required to make the examples work together as-is and is what prompted @rahul-tuli ’s investigation which led to him finding the issue with the server reading local files.

Reply:
Thank you @dbarbuzzi

Reply:
@dbarbuzzi you are right.